packages/strapi-design-system/src/Accordion/Accordion.tsx

```diff
@@ -53,95 +59,87 @@ const AccordionWrapper = styled(Box)`
   }
 `;
 
-export const Accordion = ({
-  children,
-  expanded,
-  id,
-  size,
-  variant,
-  disabled,
-  error,
-  hasErrorMessage,
-  onToggle,
-  toggle,
-}) => {
-  const generatedId = useId('accordion', id);
-
-  return (
-    <AccordionContext.Provider value={{ expanded, onToggle, toggle, id: generatedId, size, variant, disabled }}>
-      <AccordionWrapper
-        data-strapi-expanded={expanded}
-        disabled={disabled}
-        aria-disabled={disabled}
-        expanded={expanded}
-        hasRadius
-        variant={variant}
-        error={error}
-      >
-        {children}
-      </AccordionWrapper>
-      {error && hasErrorMessage && (
-        <Box paddingTop={1}>
-          <Typography variant="pi" textColor="danger600">
-            {error}
-          </Typography>
-        </Box>
-      )}
-    </AccordionContext.Provider>
-  );
-};
+export type AccordionSize = 'S' | 'M';
```

Comment:
I'm wondering if we should centralize the sizing schema somewhere, because it will always be a subset of `S|M|L`? This way we can make it clear how it works, when new components are created using the `size` prop. Wdyt?

Reply:
hmmmm we'd need a way for certain components to not use `L`.

Reply:
I guess that would be an enhancement anyways, so feel free to ignore it for now :)